src/time_zone_format.cc

```diff
@@ -63,48 +63,6 @@ char* strptime(const char* s, const char* fmt, std::tm* tm) {
 }
 #endif
 
-// Convert a cctz::weekday to a tm_wday value (0-6, Sunday = 0).
-int ToTmWday(weekday wd) {
```

Comment:
I could also save these functions and just return weekday_to_number or number_to_weekday. Then we still have these names but that did not feel right as example this function has comment about that sunday = zero. When we use weekday_to_number straight we explicitly see that sunday is zero.